tests/resources/test_data_concepts.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import unittest
+import mock
+from uuid import uuid4
+
+from citrine.resources.measurement_spec import MeasurementSpec, MeasurementSpecCollection
+from citrine.resources.condition_template import ConditionTemplate
+from taurus.entity.link_by_uid import LinkByUID
+from taurus.entity.bounds.real_bounds import RealBounds
+
+spec_uid = str(uuid4())
+measurement_spec_dict = {
+    'uids': {'id': spec_uid},
+    'name': 'spec name',
+    'tags': [],
+    'notes': None,
+    'conditions': [],
+    'parameters': [],
+    'template': None,
+    'file_links': [],
+    'type': 'measurement_spec'
+}
+
+class SessionTests(unittest.TestCase):
```

Comment:
Is there a reason we have to use unittest here, rather than sticking with pytest patterns?